packages/studioCMS/src/integrations/studioCMSDashboard/utils/webVital.ts

```diff
@@ -11,14 +12,25 @@ export type WebVitalsResponseItem = {
 };
 
 export async function getWebVitals(): Promise<WebVitalsResponseItem[]> {
-	const webVitals: WebVitalsResponseItem[] = await db
-		.select()
-		.from(AstrojsWebVitals_Metric)
-		.catch(() => {
-			return [] as WebVitalsResponseItem[];
-		});
-	if (!webVitals) {
+	// Check if AstrojsWebVitals exists
+	if (!AstrojsWebVitals_Metric) {
+		logger.warn(
+			'@astrojs/web-vitals table not found.  If you have not installed the package you can disregard this warning.'
+		);
```

Comment:
Not sure it's a good pattern to log a warning if an optional feature isn't being used.

Reply:
This one will change to a `logger.debug` in the future.  Mainly just using logs to keep track of what is breaking and not breaking between updates... cause you know logs give info on what step it is in the process